flink-runtime/src/main/java/org/apache/flink/runtime/metrics/MetricRegistry.java

```diff
@@ -219,9 +249,16 @@ public ScopeFormats getScopeFormats() {
 	public void register(Metric metric, String metricName, MetricGroup group) {
```

Comment:
In this case I get error in the compile for extending class of MetricRegistry. 

For example org.apache.flink.runtime.metrics.groups.TaskMetricGroupTest.CountingMetricRegistry#register(Metric , String , MetricGroup) and  org.apache.flink.runtime.metrics.groups.MetricGroupTest.ExceptionOnRegisterRegistry#register(Metric , String , MetricGroup)
Cause by:  method does not override or implement a method from a supertype

It is simple tests classes, I can change methods signature in their. But will not be any problems in future because of this change in MetricRegistry?